provisioner/terraform/testutil.go

```diff
@@ -0,0 +1,140 @@
+package terraform
```

Comment:
Should we name this `util_test.go` so it can't be imported from regular code?

Reply:
Hhmm I tried that but ran into an issue; I think I wanted to use `util_test` but then Go complained that there were multiple package names. Looking at this again though I should definitely change this to `terraform_test` and be done with it.

Reply:
Ah, I was simply referring to renaming the _file_ from `testutil.go` to `util_test.go` (`_test.go` will exclude it from non-test code). Up to you if you want to change the package from `terraform` to `terraform_test` (internal vs external testing).